tests/TestHelpers/TestClock.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Tuf\Tests\TestHelpers;
+
+use Tuf\Helper\Clock;
+
+/**
+ * A test clock class that uses the time from the test fixtures as starting time.
+ */
+class TestClock extends Clock
+{
+    private $time = null;
```

Comment:
```suggestion
    /**
     * The current time.
     *
     * Defaults to the same timestamp used in generate_fixtures.py to create
     * test fixtures.
     */
    private $time = 1577836800;
```

If we use this default value, we can remove __construct().